src/components/ErrorModal/ErrorModal.js

```diff
@@ -71,14 +73,16 @@ function ErrorModal(props) {
                 br: () => <br />,
               };
               break;
-            default:
+            default: {
+              values.itemLimit = parameters.find(item => item.key === ITEM_LIMIT_KEY)?.value;
```

Comment:
This will set `values.itemLimit` to `null` if there isn't a match. Of course, that shouldn't happen, but if it does, I think we want to keep a numeric value in there (zero), instead of null.

Reply:
Thank you @zburke.
If there is no match we will have `undefined`. In this case `itemLimit` parameter will not be used in the message at all.
Could you clarify what we should show if we have `itemLimit` is 0?